plugins/server-stats/frontend/app.js

```diff
@@ -54,7 +54,7 @@ const { getUserApps } = require('../../../api/utils/rights');
                             timeout: 4E3,
```

Comment:
Check on https://github.com/Countly/countly-server/pull/4460/files#diff-aefa6b585d0d6fde5db7e44f9b8dac0a96e995b5c68d9306e33eb084954d9764R12 should be fixed to check offline mode and frontend.countly_tracking setting